integration-tests/src/test/java/oracle/kubernetes/operator/ITUsabilityOperatorHelmChart.java

```diff
@@ -596,4 +581,130 @@ public void testCreateWithDefaultTargetDomainInstall() throws Exception {
     }
     logger.info("SUCCESS - " + testMethodName);
   }
+  /**
+   * Create operator and verify its deployed successfully. Create domain1 and verify domain is
+   * started. Call helm upgrade to add domainnew to manage, verify both domains are managed by
+   * operator Call helm upgrade to remove first domain from operator target domains, verify it can't
+   * not be managed by operator anymore
+   *
+   * @throws Exception
+   */
+  @Test
+  public void testAddRemoveDomainUpdateOperatorHC() throws Exception {
+    Assume.assumeFalse(QUICKTEST);
+    String testMethodName = new Object() {}.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
+    logger.info("Creating Operator & waiting for the script to complete execution");
+    // create operator
+    Map<String, Object> operatorMap = TestUtils.createOperatorMap(number, true);
+    Operator operator = new Operator(operatorMap, RESTCertType.SELF_SIGNED);
+    operator.callHelmInstall();
+    Domain domain = null;
+    Domain domainnew = null;
+    boolean testCompletedSuccessfully = false;
+    try {
+      logger.info("kubectl create namespace test" + (number + 1));
+      ExecCommand.exec("kubectl create namespace test" + (number + 1));
+      domain = createVerifyDomain(number, operator);
+      ArrayList<String> targetDomainsNS =
+          (ArrayList<String>) (operator.getOperatorMap().get("domainNamespaces"));
+      targetDomainsNS.add("test" + (number + 1));
+      upgradeVerifyOperatorDomainNamespaces(operator, targetDomainsNS);
+      domainnew = createVerifyDomain(number + 1, operator);
+      logger.info("verify that old domain is managed by operator after upgrade");
+      operator.verifyDomainExists(domain.getDomainUid());
+      logger.info("Upgrade to remove first domain");
+      targetDomainsNS.remove("test" + (number));
+      upgradeVerifyOperatorDomainNamespaces(operator, targetDomainsNS);
+      Thread.sleep(60 * 1000);
```

Comment:
i would prefer a loop to check for completion and then continue, so we do not add artificial and unnecessary delays to our test suite please.  we are trying to minimize the time taken to run tests.